plugins/elantp/fu-elantp-i2c-device.c

```diff
@@ -622,6 +628,7 @@ fu_elantp_i2c_device_init (FuElantpI2cDevice *self)
 {
 	fu_device_add_flag (FU_DEVICE (self), FWUPD_DEVICE_FLAG_INTERNAL);
 	fu_device_add_flag (FU_DEVICE (self), FWUPD_DEVICE_FLAG_UPDATABLE);
+	fu_device_add_flag (FU_DEVICE (self), FWUPD_DEVICE_FLAG_NEEDS_REBOOT);
```

Comment:
I think we need to discuss this - is the touchpad _now_ non-functional while in this state?  I guess if it's recovery mode it was probably _already_ non-functional, but want to confirm.  Otherwise we have a problem for GUI clients...

Reply:
It was already dead. In i2c mode it's essentially a dumb bit of glass that doesn't "do" anything.